### PR TITLE
Script Design Audit - Last modified by & last modified at 

### PR DESCRIPTION
**Pull request by Suyash-d**
**Description**
Store the username and timestamp of the last modification of a script. The DES_SCRIPT table should be extended with two new colums:
* LAST_MODIFIED_BY: 
  * description: the username of the user that made the last modification. This information is retrieved from the currently authenticated user when modification is performed via the REST API. Should the script be updated from the command line then the default 'admin' account is used
  * type: string (255)
  * nullable: true. The value is null if the current script has never been modified yet
* LAST_MODIFIED_AT: 
  * the timestamp of the last modification
  * type: string (255)
  * nullable: true. The value is null if the current script has never been modified yet

This information is retrieved from the currently authenticated user when the REST API is being performed.

**Tasks**
1. a
2. alter table script for existing data models

### CODE REVIEW

**Comment by BerrevoetsRobbe** [comment]
@Suyash-d could you move the last modified information to script version instead of script?

**Comment by BerrevoetsRobbe** [comment]
@Suyash-d could you move the created information to script version instead of script?

File: core/conf/application-metadata.yml [hunk 1]
```diff
@@ -1785,7 +1785,7 @@ iesi:
             defaultTimestamp:
           LOAD_TMS:
             description: Load timestamp indicating when the record was inserted in the table
-            order: 5
+            order: 7
```
Comment (Suyash-d) [inline core/conf/application-metadata.yml:1788]: Remove Order

File: core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/script/ScriptConfiguration.java [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
     private static final String INSERT_QUERY = "INSERT INTO " +
             MetadataTablesConfiguration.getInstance().getMetadataTableNameByLabel("Scripts").getName() +
-            " (SCRIPT_ID, SECURITY_GROUP_ID, SECURITY_GROUP_NAME, SCRIPT_NM, SCRIPT_DSC) VALUES " +
+            " (SCRIPT_ID, SECURITY_GROUP_ID, SECURITY_GROUP_NAME, SCRIPT_NM, SCRIPT_DSC ) VALUES " +
```
Comment (Suyash-d) [inline core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/script/ScriptConfiguration.java:60]: Make it as it was

File: core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/script/Script.java [hunk 3]
```diff
@@ -27,6 +27,7 @@
     private List<Action> actions;
     private List<ScriptLabel> labels;
 
+
```
Comment (Suyash-d) [inline core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/script/Script.java:30]: Remove line

File: core/java/iesi-core/src/test/java/io/metadew/iesi/metadata/configuration/script/ScriptBuilder.java [hunk 4]
```diff
@@ -76,6 +75,7 @@ public ScriptBuilder numberOfLabels(int numberOfLabels) {
         return this;
     }
 
+
```
Comment (Suyash-d) [inline core/java/iesi-core/src/test/java/io/metadew/iesi/metadata/configuration/script/ScriptBuilder.java:78]: Remove line

File: core/java/iesi-core/src/test/java/io/metadew/iesi/metadata/configuration/script/ScriptVersionConfigurationTest.java [hunk 5]
```diff
@@ -176,4 +182,6 @@ void scriptGetAllVersionsOfScriptMultipleTest() {
         assertEquals(Stream.of(scriptVersion1, scriptVersion2).collect(Collectors.toList()), scriptVersions);
     }
 
+
```
Comment (Suyash-d) [inline core/java/iesi-core/src/test/java/io/metadew/iesi/metadata/configuration/script/ScriptVersionConfigurationTest.java:185]: Remove line

**Review by BerrevoetsRobbe — COMMENTED** [review]
approved

**Review by BerrevoetsRobbe — COMMENTED** [review]
approved

**Pull request merged**